virtualizarr/tests/test_manifests/test_array.py

```diff
@@ -124,13 +124,43 @@ def test_partly_equals(self): ...
 
 
 class TestBroadcast:
+    def test_broadcast_existing_axis(self):
+        marr = create_manifestarray(shape=(1, 2), chunks=(1, 2))
+        expanded = np.broadcast_to(marr, shape=(3, 2))
+        assert expanded.shape == (3, 2)
+        assert expanded.chunks == (1, 2)
+        assert expanded.manifest.dict() == {
+            "0.0": {"path": "file.0.0.nc", "offset": 0, "length": 5},
+            "1.0": {"path": "file.0.0.nc", "offset": 0, "length": 5},
+            "2.0": {"path": "file.0.0.nc", "offset": 0, "length": 5},
+        }
+
+    def test_broadcast_new_axis(self):
```

Comment:
Note to self: this is still buggy and needs more rigorous testing to ensure the shapes of arrays / chunks are correct. I should add some hypothesis property-based tests to bulletproof this.

Reply:
Adding hypothesis tests turned out to be a big rabbit whole so I just added some parametrized tests instead.